index.html

```diff
@@ -14,13 +14,72 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@500&display=swap" rel="stylesheet">
 </head>
-<body>
-  <header>
+<body class="background-color-gray">
+  <header class="background-color-purple text-white">
+    <div class="container opensans">
```

Comment:
I would suggest just applying `opensans` to the `body` element and letting the text in the document inherit the styling from there, rather than applying it to each text element.